pep-0646.rst

```diff
@@ -24,6 +24,14 @@ in numerical computing libraries such as NumPy and TensorFlow to be
 parameterised with the array *shape*, enabling static type checkers
 to catch shape-related bugs in code that uses these libraries.
 
+Acceptance
+==========
+
+This PEP was accepted for Python 3.11, with the caveat that details around
```

Comment:
Should we link to the acceptance message here?

Reply:
The link should go in a `Resolution` header.